_includes/menu.html

```diff
@@ -43,12 +43,12 @@ <h2 id="advanced-documentation">Advanced Documentation</h2>
 <h2 id="community-and-contributions">Community and Contributions</h2>
 
 <ul class="list-unstyled">
-	<li><a href="https://github.com/OrleansContrib">Community add-ons to Orleans</a></li>
-	<li><a href="https://github.com/OrleansContrib/meetups">Virtual Meetups</a></li>
-	<li><a href="https://gitter.im/dotnet/orleans">Gitter Chat Forum</a></li>
-	<li><a href="https://twitter.com/projectorleans">Twitter</a></li>
-	<li><a href="https://orleans.codeplex.com">Orleans on Codeplex</a></li> 
-	<li><a href="https://orleans.codeplex.com/discussions">Discussion forum</a></li>
+	<li><a href="https://github.com/OrleansContrib">OrleansContrib - Community add-ons to Orleans</a></li>
+	<li><a href="https://github.com/OrleansContrib/meetups">Orleans Community Virtual Meetups</a></li>
+	<li><a href="https://gitter.im/dotnet/orleans">Orleans Gitter Chat Forum</a></li>
+	<li><a href="https://twitter.com/projectorleans">Twitter - @ProjectOrleans</a></li>
+	<li><a href="https://orleans.codeplex.com">Orleans on Codeplex (DEPRECATED)</a></li> 
+	<li><a href="https://orleans.codeplex.com/discussions">CodePlex Discussion Forum (DEPRECATED)</a></li>
```

Comment:
So remove the Discussion Forum link because we now use GH for that too? Just leave link to CP home page [and mark "historical"].